apps/aecore/src/aec_peers.erl

```diff
@@ -787,10 +787,15 @@ next_connect_delay(State) ->
     case is_outbound_allowed(State) of
         false  -> infinity;
         true ->
-            #state{ last_connect_time = LastTime, outbound = Outbound } = State,
-            ExpDelay = floor(math:pow(2, Outbound - 1)) * 1000,
-            BoundDelay = min(ExpDelay, ?MAX_CONNECTION_INTERVAL),
-            max(?MIN_CONNECTION_INTERVAL, BoundDelay - (timestamp() - LastTime))
+            case aeu_env:user_config([<<"sync">>, <<"peer_analytics">>], false) of
```

Comment:
Whenever there is a schema default (and there should be), I think it should be honored.
```
{ok, Bool} = aeu_env:find_config([<<"sync">>, <<"peer_analytics">>],
                                 [user_config, schema_default, {value,false}]),
case Bool of
...
```